htdocs/ticket/class/ticket.class.php

```diff
@@ -865,6 +870,11 @@ public function update($user = 0, $notrigger = 0)
 		global $conf, $langs, $hookmanager;
 		$error = 0;
 
+		if (empty($this->oldcopy) && empty($notrigger)) {
```

Comment:
Having the cloning (with all sql that this implied) done into a low level method that should only update data that are already set by caller looks a bad practive.
According to modulebuilder template, the oldcopy should be set ust before calling the ->update()
with
`$object->oldcopy = dol_clone($object);`